core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/MetricNameService.java

```diff
@@ -33,28 +33,45 @@
  */
 public class MetricNameService implements MetricFilter {
 
-    public static final String TENANT_ID = "admin";
 
     private static final String separator = ":";
 
-    private String hostname;
+    private final String adminTenant;
+    private final String hostname;
 
     public MetricNameService() {
         try {
             hostname = InetAddress.getLocalHost().getHostName();
         } catch (UnknownHostException e) {
             throw new RuntimeException("Failed to get hostname", e);
         }
+
+        this.adminTenant = "admin";
```

Comment:
I just realized that my previous comment is based on changes in a branch I am working on. Sorry for the confusion. In master, the no-arg constructor is only used in tests; however, that is going to change.